tests/test_rise.py

```diff
@@ -57,7 +57,7 @@ def test_rise_text(self):
         review = 'such a bad movie'
         expected_words = ['such', 'a', 'bad', 'movie']
         expected_word_indices = [0, 5, 7, 11]
-        expected_positive_scores = [0.3295266, 0.3521292, 0.023648001, 0.3347813]
+        expected_positive_scores = [0.3044678, 0.28736606, 0.03623142, 0.23650846]
```

Comment:
Seems like the general trend in the values has remained unchanged. That's great!